azerothcore/azerothcore-wotlk
# fix(DB/Creature): removes Elixir of Dream Vision from creature loot

## Changes Proposed:
-  Removes Elixir of Dream Vision (9197) from the possible loot of 10 mobs

## Issues Addressed:

- Closes #7989
- Closes https://github.com/chromiecraft/chromiecraft/issues/1763

## SOURCE:


## Tests Performed:

- Done the tests described below


## How to Test the Changes:


1. Run SQL query - Should change 10 rows
2. Check that no mob drops the item - ``` SELECT * FROM `creature_loot_template` WHERE `Item` = 9197; ``` should return 0

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
